src/openms_ml/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+# Copyright The OpenMS Team -- Eberhard Karls University Tuebingen,
+# ETH Zurich, and Freie Universitaet Berlin 2002-2022.
+#
+# This software is released under a three-clause BSD license:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of any author or any participating institution
+#    may be used to endorse or promote products derived from this software
+#    without specific prior written permission.
+# For a full list of authors, refer to the file AUTHORS.
+# --------------------------------------------------------------------------
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL ANY OF THE AUTHORS OR THE CONTRIBUTING
+# INSTITUTIONS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+# OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+# OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+# ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# --------------------------------------------------------------------------
+# $Maintainer: Timo Sachsenberg $
+# $Authors: Timo Sachsenberg $
+# --------------------------------------------------------------------------
+
+project("OpenMS_ML")
+cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
+
+#set(CMAKE_AUTOUIC OFF) # leave off since this is broken, see qt5_wrap_ui below
+#set(CMAKE_AUTOMOC ON)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# --------------------------------------------------------------------------
+# include custom macros for openms_gui lib
+# TODO check if needed... include(add_mac_bundle.cmake)
+
+# --------------------------------------------------------------------------
+# big include file for headers and cpp files
+include (${PROJECT_SOURCE_DIR}/includes.cmake)
+
+#------------------------------------------------------------------------------
+# add the library
+openms_add_library(TARGET_NAME OpenMS_ML
+                   SOURCE_FILES ${OpenMSML_sources}
+                   HEADER_FILES ${OpenMSML_sources_h}
+                   INTERNAL_INCLUDES ${PROJECT_SOURCE_DIR}/include ${PROJECT_BINARY_DIR}/include
+                   EXTERNAL_INCLUDES
+                   LINK_LIBRARIES ${OpenMS_ML_DEP_LIBRARIES}
+                   DLL_EXPORT_PATH "OpenMS/") # TODO ???
```

Comment:
Not sure what should be here

Reply:
Will be the folder where TARGETNAMEConfig.h goes for the import/export TARGETNAME_DLL_API macros.
Should be the (include/)OpenMS_ML folder or how you will call it.